docs/Contribute.ipynb

```diff
@@ -170,6 +170,43 @@
     "![Fetch Upstream Fork](images/fetch_upstream_fork.png)  "
    ]
   },
+  {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+     ":::{admonition} Instructions for Pulling Main into a Development Branch\n",
+     ":class: toggle\n",
+     "1. Open a terminal\n",
+     "2. Change the current working directory to your local development repository (e.g., cd Git/stumpy_dev.git)\n",
+     "3. Check out your local development branch (e.g., git switch some_new_feature)\n",
+     "4. Commit all changes in your local development branch (e.g., git add some_file.py and git commit)\n",
+     "5. Fetch the branches and their respective commits from the upstream repository (e.g., git fetch upstream)\n",
+     "6. Check out your fork's local default branch (e.g., git checkout main)\n",
+     "7. Merge the changes from the upstream default branch - in this case, upstream/main - into your local default branch (e.g., git merge upstream/main). This brings your fork's default branch into sync with the upstream repository, without losing your local changes.\n",
+     "8. If your local main branch didn't have any unique commits, Git will perform a fast-forward. Otherwise, if your local main branch had unique commits, you may need to resolve conflicts. Note that this does not affect your development branch!\n",
+     "9. Next, switch over to your development branch (e.g., git switch some_new_feature)\n",
+     "10. Finally, merge the main branch into your development branch (e.g., git merge main)\n",
+     "    - You may see something like the following, if you do, you will need to open up the files tagged with CONFLICT and resolve the merge conflicts. Once that's done, you will need to commit the changes and push the commit (e.g., git push) back to Github.\n",
+     "\n",
+     "```",
+     "git merge main\n",
+     "Auto-merging stumpy/aamp_stimp.py\n",
+     "Auto-merging stumpy/core.py\n",
+     "Auto-merging stumpy/mpdist.py\n",
+     "CONFLICT (content): Merge conflict in stumpy/mpdist.py\n",
+     "Auto-merging stumpy/mstumped.py\n",
+     "CONFLICT (content): Merge conflict in stumpy/mstumped.py\n",
+     "Auto-merging stumpy/ostinato.py\n",
+     "Auto-merging stumpy/stimp.py\n",
+     "CONFLICT (content): Merge conflict in stumpy/stimp.py\n",
+     "Auto-merging stumpy/stumped.py\n",
+     "CONFLICT (content): Merge conflict in stumpy/stumped.py\n",
+     "Automatic merge failed; fix conflicts and then commit the result.\n",
+     "You will need to open up the files tagged with CONFLICT and resolve the merge conflicts. Once that's done, you will need to commit the changes and push the commit (e.g., git push) back to Github.\n",
+     "\n",
```

Comment:
@joehiggi1758 This doesn't look quite right. I think you still need to CLOSE the triple backticks (you've only initialized them above on line 191).

Reply:
Shoot I'm on it!

Reply:
@seanlaw bit stuck here - I've tried a bunch of variations but the code cell keeps rendering as follows when I add the closing ticks:

![image](https://github.com/user-attachments/assets/f01c2eda-bef6-4c60-8f35-55c86f7cefa0)

Reply:
This worked for me:

````
:::{admonition} Instructions for Pulling Main into a Development Branch
:class: toggle
1. Open a terminal
2. Change the current working directory to your local development repository (e.g., cd Git/stumpy_dev.git)
3. Check out your local development branch (e.g., git switch some_new_feature)
4. Commit all changes in your local development branch (e.g., git add some_file.py and git commit)
5. Fetch the branches and their respective commits from the upstream repository (e.g., git fetch upstream)
6. Check out your fork's local default branch (e.g., git checkout main)
7. Merge the changes from the upstream default branch - in this case, upstream/main - into your local default branch (e.g., git merge upstream/main). This brings your fork's default branch into sync with the upstream repository, without losing your local changes.
8. If your local main branch didn't have any unique commits, Git will perform a fast-forward. Otherwise, if your local main branch had unique commits, you may need to resolve conflicts. Note that this does not affect your development branch!
9. Next, switch over to your development branch (e.g., git switch some_new_feature)
10. Finally, merge the main branch into your development branch (e.g., git merge main)
    - You may see something like the following, if you do, you will need to open up the files tagged with CONFLICT and resolve the merge conflicts. Once that's done, you will need to commit the changes and push the commit (e.g., git push) back to Github.

```
git merge main
Auto-merging stumpy/aamp_stimp.py
Auto-merging stumpy/core.py
Auto-merging stumpy/mpdist.py
CONFLICT (content): Merge conflict in stumpy/mpdist.py
Auto-merging stumpy/mstumped.py
CONFLICT (content): Merge conflict in stumpy/mstumped.py
Auto-merging stumpy/ostinato.py
Auto-merging stumpy/stimp.py
CONFLICT (content): Merge conflict in stumpy/stimp.py
Auto-merging stumpy/stumped.py
CONFLICT (content): Merge conflict in stumpy/stumped.py
Automatic merge failed; fix conflicts and then commit the result.
You will need to open up the files tagged with CONFLICT and resolve the merge conflicts. Once that's done, you will need to commit the changes and push the commit (e.g., git push) back to Github.
```
:::
````

Reply:
Ahhh - got it working, I was editing the notebook as a text file, should've just opened it as a notebook in the first place!